arm-compute/compositeComputeClient.json

```diff
@@ -4,7 +4,7 @@
     "description": "Composite Swagger for Compute Client"
   },
   "documents": [
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2016-03-30/swagger/compute.json",
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2016-03-30/swagger/containerService.json"
+    "./arm-compute/2016-03-30/swagger/compute.json",
+    "./arm-compute/2016-03-30/swagger/containerService.json"
   ]
```

Comment:
Per the discussion for Issue #177 @devigned and @lmazuel what is the final decision regarding the use of relative paths?